monitor/grafana/zeebe.json

```diff
@@ -9349,6 +9349,16 @@
               "interval": "30s",
               "legendFormat": "{{le}}",
               "refId": "A"
+            },
+            {
+              "datasource": {
+                "uid": "$DS_PROMETHEUS"
+              },
+              "expr": "sum(increase(zeebe_gateway_request_latency_seconds_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",partition=~\"$partition\"}[$__rate_interval])) by (le)",
```

Comment:
:thought_balloon:  have you tried with :

```
sum(increase(zeebe_gateway_request_latency_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",partition=~\"$partition\"}[$__rate_interval])) by (le) or sum(increase(zeebe_gateway_request_latency_seconds_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",partition=~\"$partition\"}[$__rate_interval])) by (le)

```

Reply:
I did, and it kept showing me weird values - when both series return something over the same window of time, only one of them is selected :thinking: